axum/src/handler/mod.rs

```diff
@@ -220,13 +221,13 @@ all_the_tuples!(impl_handler);
 /// A [`Service`] created from a [`Handler`] by applying a Tower middleware.
 ///
 /// Created with [`Handler::layer`]. See that method for more details.
-pub struct Layered<L, H, T, S, B> {
+pub struct Layered<L, H, T, S, B, B2> {
```

Comment:
How many type params are too many 🙃 Couldn't find a way to make it work with adding `B2`.